index.html

```diff
@@ -829,24 +829,29 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
       <section id="solid-ldp-acls">
         <h2>ACLs are LDP RDF Sources</h2>
         <p>
-          The linked Access Control List Resource (ACL) for a controlled resource by a conforming server MUST itself be
+          The Access Control List Resource (ACL) for a controlled resource on conforming server MUST itself be
           an LDPRS. This ACL resource SHOULD be located in the same server as the controlled resource.
         </p>
       </section>
       <section id="link-rel-acl">
         <h2>ACLs are discoverable via Link Headers</h2>
         <p>
-          A conforming server MUST advertise the linked Access Control List Resource (ACL) for a controlled resource in
-          HTTP responses with a <code>Link: rel="acl"</code> header. This interaction pattern is described in [[!WEBAC]]
-          and [[!SOLIDWEBAC]].
+          A conforming server MUST advertise the appropriate Access Control List Resource (ACL) for every controlled
+          resource in HTTP responses with a <code>Link: rel="acl"</code> header. This interaction pattern is described
+          in [[!SOLIDWEBAC]].
```

Comment:
Recommend adding a direct link to: https://github.com/solid/web-access-control-spec#acl-resource-location-discovery